odcockpit/include/plugins/environmentviewer/EnvironmentViewerGLWidget.h

```diff
@@ -107,6 +107,13 @@ class SelectableNodeDescriptor;
                     virtual void drawScene();
 
                 private:
+                    void drawOneCPCPointNoIntensity(const uint16_t &distance_integer, const float &azimuth, const float &verticalAngle, const uint8_t &distanceEncoding);
+                    void drawOneCPCPointWithIntensity(const uint16_t &distance_integer, const float &azimuth, const float &verticalAngle, const uint8_t &distanceEncoding, const uint8_t &numberOfBitsForIntensity, const uint8_t &intensityPlacement, const uint16_t &mask, const float &intensityMaxValue);
+                    void drawCPC32noIntensity(const uint8_t &part, const uint8_t &entriesPerAzimuth, const float &startAzimuth, const float &endAzimuth, const uint8_t &distanceEncoding);
+                    //part == 1: the first CPC of a HDL-32E scan with 12 layers
```

Comment:
@HangYin2015: You should add a short explanation about the encoding scheme (so that we know in a year from now why we send the CPCs as we do ;-)